File: MarsRoverProb/src/com/marsroverprob/commands/RotateRightCommand.java
package com.marsroverprob.commands;

import com.marsroverprob.helper.RoverHelper;
import com.marsroverprob.model.Rover;

public class RotateRightCommand implements RoverCommand {

  public void execute(Rover rover, String maxPos) {
    RoverHelper.rotateRight(rover, maxPos);
  }

}

File: MarsRoverProblem_C#/Source/marsroverprob/commands/RotateRightCommand.cs
using System;

using marsroverprob.commands.intf;
using marsroverprob.model;
using marsroverprob.helper;

namespace marsroverprob.commands.rotateright 
{
    class RotateRightCommand : IRoverCommand {
        public void execute(Rover rover, String maxPos) {
            RoverHelper.rotateRight(rover, maxPos);
        }
    }
}